scrapers/koji_scraper.py

```diff
@@ -7,6 +7,7 @@
 import logging
 
 log = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Can you put this in `scripts/scraper.py` instead? That way all scrapers that are run through `scraper.py` will log as debug.

Reply:
Sure, Done!